.github/workflows/release.yml

```diff
@@ -556,4 +556,32 @@ jobs:
       - name: Sync chain ${{ matrix.chain }}
         run: |
           chmod +x target/release/astar-collator
-          ./scripts/sync.sh ${{ matrix.chain }}
+          ./scripts/sync-smoke.sh ${{ matrix.chain }}
+
+  zombienet-smoke:
+    needs: native-linux
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        chain: [ "astar-dev", "shiden-dev", "shibuya-dev" ]
+
+    steps:
+      - name: Checkout the source code
+        uses: actions/checkout@v4
+
+      - name: Download pre-built collator binary
+        uses: actions/download-artifact@v3
+        with:
+          name: astar-ubuntu-latest-x86_64
+          path: target/release
```

Comment:
Shouldn't this code snippet be re-used instead?

```yml
    - name: Download pre-built collator binary
      uses: actions/download-artifact@v3
      with:
        name: astar-${{ matrix.os }}-latest-${{ matrix.arch }}
        path: ${{ matrix.os }}-${{ matrix.arch }}-bin

    - name: Make binary executable and tar gzip
      run: |
        cd ${{ matrix.os }}-${{ matrix.arch }}-bin
        chmod +x astar-collator
        tar zcvf astar-collator.tar.gz astar-collator
```
(replacing os & arch of course)

Otherwise old release binary is downloaded, unrelated to this whole run.

Reply:
no, I need just linux-x86.
Artefacts are from the current workflow.